draft-looker-jwt-cwt-status-list.md

```diff
@@ -32,38 +32,57 @@ This specification defines a status list representation and processing rules for
 
 The following diagram depicts the basic conceptual relationship.
 
 ~~~ ascii-art
 
 +---------------+                     +---------------+
-|               |                     |               |
-|               |                     |               |
-|     Token     |    References       |  Status List  |
-|(JWT/CWT Based)|-------------------->|(JWT/CWT Based)|
-|               |                     |               |
-|               |                     |               |
+|               |      References     |               |
+|               |-------------------->|               |
+| Subject Token |                     |  Status List  |
+|(JWT/CWT Based)|                     |(JWT/CWT Based)|
+|               |  Describes Status   |               |
+|               |<--------------------|               |
 +---------------+                     +---------------+
 
 ~~~
 
+## Rationale
+
+Revocation mechanisms are an essential part for most identity ecoosystems. In the past, revocation of X.509 TLS certificates has been proven difficult as traditional certificate revocation lists (CRLs) have limited scalability and the Online Certificate Status Protocol (OCSP) has additional privacy risks as the client is leaking the requested website to a third party. OSCP stapling is adressing some of these problems at the cost of less up-to-date data. Modern approaches use accumulator-based revocation registries and Zero-Knowledge-Proofs to accomodate for this privacy gap but face scalability issues again.
+
+The approach of this specification seeks to find a balance between scalability, security and privacy by minimizing the status information to mere bits and compressing the resulting binary data. Thereby a Status List may contain statuses of 100.000 or more Subject Tokens, but still remain relatively small. Placing large amounts of Subject Tokens into the same list also enables a herd privacy towards the Issuer or Status List Provider.
+
+
 # Conventions and Definitions
 
 {::boilerplate bcp14-tagged}
 
+# Terminology
+
+Status List
+ A token in JWT or CWT representation that expresses the statuses of many Subject Tokens.
+
+Subject Token
+ A token in JWT or CWT representation which contains a reference to a Status List. The information from the Stataus List may give a verifier additional information about up-to-date status of the Subject Token.
+
+Status List Provider
+ An entity that hosts the Status List as a resource for potential verifiers. The Status List Provider may be the issuer of the Status List but may also be outsourced to a trusted third party.
```

Comment:
Not super convinced of this, I think this is pending resolution around whether we agree if the status list provider != issuer otherwise I think issuer suffices?

Reply:
This relates to #10 and is in line with the issuer-holder-verifier model described in the SD-JWT spec